payload-schemas/schemas/common/installation.schema.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "common/installation.schema.json",
+  "type": ["object", "null"],
```

Comment:
I just noticed these - I think it would be better to use `oneOf` at the reference site to mark possible nulls rather than to have this `null` here.

My reasoning is that right now the type is this:

```

type GithubInstallation= { ... } | null;

interface MyPayload {
  installation: GithubInstallation;
}
```

That means theres no way to reference the object type for `Installation` without always doing some extra work:

```
const describeInstallation = (installation: Exclude<GithubInstallation, null>) => {
  // 
};
```

Even if the property could be `null` everywhere the type is used, it's more to indicate that at the property-level rather than the type-level.

Reply:
makes sense, can you make the change?

Reply:
Yup, will do tomorrow.